test-psql2mysql.py

```diff
@@ -28,7 +28,7 @@
 p2m = importlib.import_module("psql2mysql")
 
 
-class FakeColumn():
+class FakeColumn(object):
```

Comment:
only needed in Python2 not in Python3, but I guess that this will run in both, so is OK